docs/design/fl-parallelism/sw-availability-data-propagation.puml

```diff
@@ -0,0 +1,64 @@
+@startuml
+title Switch availability info propagation
+
+participant FL0
```

Comment:
Zero on the pic looks like capital `o`. Maybe we can call it `FL-0` or `FL1` for better readability?

Reply:
replaced with 1